scipy/stats/tests/test_morestats.py

```diff
@@ -1636,6 +1636,15 @@ def test_auto(self):
         d = np.arange(1, 52)
         assert_equal(stats.wilcoxon(d), stats.wilcoxon(d, mode="approx"))
 
+    @pytest.mark.parametrize('size', [3, 5, 10])
```

Comment:
is any testing required for the new keywords `axis`, `keepdims` and `nan_policy`?

Reply:
They aren't new, actually. They were added by the `_axis_nan_policy` decorator on `wilcoxon` a while ago, and they are very thoroughly tested in `test_axis_nan_policy.py`. Funny thing though,  - the `axis` argument is still being completely handled by the decorator! The only time the new vectorized behavior is used is when called with an array by the permutation method. I need to change that!

Reply:
thanks for the clarification, I saw that you added `axis` in commit 3d87f985cb848e8343a484ec1e13c47414ef8fe5

Reply:
Yeah, now the native vectorization can actually be used : )